.github/workflows/upload-and-deploy.yaml

```diff
@@ -46,13 +47,162 @@ jobs:
         run: echo "NAMESPACE=$GITHUB_REF_NAME" >> $GITHUB_ENV
 
       - name: Copy files to templates bucket, use dev cloudformation bucket
-        run: python src/scripts/manage_artifacts/artifacts.py --upload --namespace $NAMESPACE --cfn_bucket ${{ vars.CFN_BUCKET }}
+        run: >
+          python src/scripts/manage_artifacts/artifacts.py
+          --upload
+          --namespace $NAMESPACE
+          --cfn_bucket ${{ vars.CFN_BUCKET }}
+
+      - name: Copies over test files from ingestion bucket
+        run: >
+          aws s3 sync s3://recover-dev-ingestion/ s3://$DEV_INPUT_DATA_BUCKET/$NAMESPACE/
```

Comment:
We should move the pilot data to a specific prefix in the ingestion bucket (`s3://recover-dev-ingestion/pilot-data`? I'll leave the naming convention up to you). Digital health can also write to this bucket so we want to keep the behavior of this line predictable.